libkbfs/keybase_service_base.go

```diff
@@ -681,3 +684,19 @@ func (k *KeybaseServiceBase) GetPublicCanonicalTLFNameAndID(
 
 	return res, nil
 }
+
+// EstablishMountDir asks the service for the current mount path
+func (k *KeybaseServiceBase) EstablishMountDir(ctx context.Context) (
+	string, error) {
+	dir, err := k.kbfsMountClient.GetCurrentMountDir(ctx)
+	if err != nil || dir == "" {
```

Comment:
I'm worried that `err` could be something fatal, like "You don't have permission to connect to the service" or "This is not a Windows system so clearly you are trying to run a Windows binary on the wrong system".  If there's a specific error that should be treated non-fatally, you should check for it explicitly.